SPECS-EXTENDED/libvpx/libvpx.spec

```diff
@@ -212,10 +214,13 @@ rm -rf %{buildroot}%{_prefix}/src
 
 %ldconfig_scriptlets
 
+%check
+make test
+
 %files
 %license LICENSE
 %doc AUTHORS CHANGELOG README
-%{_libdir}/libvpx.so.%{somajor}*
```

Comment:
Non-blocker: can we keep this line unchanged? Makes it clearer that `somajor` is directly linked to the major version of the shared library.